vyper/compiler.py

```diff
@@ -20,8 +16,7 @@ def gas_estimate(origcode, *args, **kwargs):
     if code.value == 'seq':
         if code.args[-1].value == 'return':
             code = code.args[-1].args[1].args[0]
-    else:
-        code = code.args[1].args[0]
```

Comment:
No other LLL statement ends up here except `seq`?

Reply:
Correct, I think it was some legacy case that needed it. There was no scenario that I could find that needed it.